### ISSUE TITLE
Measure the amount of clicks on the enroll button for not connected LMS using web analytics

### ISSUE THREAD

**Issue by igobranco** [openfun/richie#1592]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
Measure the amount of clicks on the enroll button, on the course page, for courses that aren't connected to any LMS.

**Describe the solution you'd like**
On the template `src/richie/apps/courses/templates/courses/cms/fragment_course_run.html` on the anchor with the class `course-run-enrollment__cta`, onclick of the anchor send a web analytics event to the provider (eg. Google Analytics).
With this method we can measure how many clicks, potencial enrollments, that we have in general and per org, course and course run (using the web analytics custom dimensions).

**Do you want to work on it through a Pull Request?**
Yes, I will send a PR.

**Comment by sampaccoud** [comment]
@igobranco wasn't this done in 2.14?

**Comment by igobranco** [comment]
No. It is sending events when we use the frontend react code to enroll the students on the LMS.
We would need at some point in future to include this feature when the course run link is not connected to any LMS.

**Comment by igobranco** [comment]
Closing old issue.
If need on NAU we can raise an event using Google Tag Manager.
